sig-api-machinery/charter.md

```diff
@@ -0,0 +1,48 @@
+# SIG API Machinery Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG API Machinery is responsible for the development and enhancement of Kubernetes master node.  The scope covers API server, persistence layer (etcd), controller manager, cloud controller manager, CustomResourceDefinition, scheduler and webhooks.
+
+### In scope
+
+#### Code, Binaries and Services
+
+All aspects of API server, API registration and discovery, generic API CRUD semantics, admission control, encoding/decoding, conversion, defaulting, persistence layer (etcd), OpenAPI, CustomResourceDefinition, webhooks, garbage collection, namespace lifecycle, and client libraries.
```

Comment:
@lavalamp Which parts of controller manager should be listed here? The informer libraries, at least, presumably (which I think we consider as distinct from the published client libraries?).

Also, this would be more readable as a bullet list.